src/components/Image.tsx

```diff
@@ -26,7 +26,7 @@ export const getImageFromList = (images: ImageProps[] = [], name?: string | Imag
 
 export const Image = ({ image, src, ...attribs }: ImageComponentProps) => {
   if (!image) {
-    return;
+    return null;
```

Comment:
Nice, we should always return `null` from components when not rendering content 👍